### PR TITLE
2894 Add api docs for accounting

### PR DESCRIPTION
**Pull request by halink0803**

### CODE REVIEW

File: accounting/apidocs/source/includes/_reserve_listed_tokens.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+                }
+            ]
+        }
+    ]
+}
+```
+
+### HTTP request
+
+`GET http://gateway.local/transactions`
```
Comment (favadi) [inline accounting/apidocs/source/includes/_reserve_listed_tokens.md]: it is not the correct URL, right? this is listed tokens, not transactions.

File: accounting/apidocs/source/includes/_reserve_transactions.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+            "isError": "0",
+            "timestamp": 1554107172000
+        }
+    ]
+}
+```
+
+### HTTP request
+
+`GET http://gateway.local/transactions`
```
Comment (favadi) [inline accounting/apidocs/source/includes/_reserve_transactions.md]: I think I see the duplicated transactions endpoint everywhere.

File: accounting/apidocs/source/includes/_reserve_listed_tokens.md (hunk 1 above)
Comment (halink0803) [reply accounting/apidocs/source/includes/_reserve_listed_tokens.md]: Yes, copy and paste issue. Fixed.

File: accounting/apidocs/source/includes/_reserve_transactions.md (hunk 2 above)
Comment (halink0803) [reply accounting/apidocs/source/includes/_reserve_transactions.md]: This one is correct.

**Pull request merged**